### PR TITLE
add Dead letter queue for stream listeners

### PR DESCRIPTION
**Pull request by blabadi**
remove spring reactive cloud stream (deprecated)
rename all properties in yaml in camel case

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.1.4.RELEASE</version>
+        <version>2.1.5.RELEASE</version>
```
Comment (d8660091) [inline pom.xml]: NIce to keep up to date!

File: ci-cd/Jenkinsfile.groovy [hunk 2]
```diff
@@ -122,28 +122,6 @@ spec:
             }
         }
 
-       /*
```
Comment (d8660091) [inline ci-cd/Jenkinsfile.groovy]: Less is better

File: maestro-app/src/main/java/bio/overture/maestro/app/infra/config/properties/DefaultApplicationProperties.java [hunk 3]
```diff
@@ -40,45 +40,45 @@
 
     final static String MAESTRO_PREFIX = "maestro";
 
-    @Value("${maestro.elasticsearch.cluster_nodes}")
```
Comment (d8660091) [inline maestro-app/src/main/java/bio/overture/maestro/app/infra/config/properties/DefaultApplicationProperties.java]: It's a little confusing to have both @ConfigurationProperties and @Value, isn't it? We can make nested class to wrap those nested values
https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-typesafe-configuration-properties

Comment (blabadi) [reply maestro-app/src/main/java/bio/overture/maestro/app/infra/config/properties/DefaultApplicationProperties.java]: at the time i didn't want to spend the time building classes just for that, done now

**Comment by d8660091** [comment]
Good job

**Pull request merged**